src/main/java/racingcar/domain/car/Car.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain.car;
+
+import racingcar.strategy.MoveStrategy;
+
+public class Car {
+    private final Name name;
+    private final Position position;
+
+    public Car(String name) {
+        this.name = new Name(name);
+        this.position = new Position(0);
+    }
+
+    private Car(Name name, int position) {
+        this.name = name;
+        this.position = new Position(position);
+    }
```

Comment:
저는 부생성자를 사용하게 되는 경우엔 모든 필드를 가진 생성자를 호출하도록 하고 있어요.

```suggestion
    private Car(Name name, Position position) {
        this.name = name;
        this.position = position;
        // 만약 유효성 검증이 필요하다면 이곳에서 진행합니다.
    }

    public Car(String name) {
        this(new Name(name), new Position(0));
    }
```

유효성 검사를 기본 생성자에서 진행하고, 모든 생성자가 그 생성자 메소드만 사용하게 한다면 객체가 어떤 생성자를 이용하더라도 유효성이 보장될거라고 생각해요 🙂